crates/python/qcs_sdk/__init__.pyi

```diff
@@ -201,16 +209,16 @@ class RegisterMatrix:
     Values in a 2-dimensional ``ndarray`` representing the final shot value in each memory reference across all shots.
     Each variant corresponds to the possible data types a register can contain.
 
-    Variants:
-        ``integer``: Corresponds to the Quil `BIT`, `OCTET`, or `INTEGER` types.
-        ``real``: Corresponds to the Quil `REAL` type.
-        ``complex``: Registers containing complex numbers.
+    ## Variants:
+    - ``integer``: Corresponds to the Quil `BIT`, `OCTET`, or `INTEGER` types.
```

Comment:
the ticks weren't changed, but I'm confused about how many are needed in this new scheme. have both single and double on this line. Then, the same line in the module below is all double ticks

Reply:
With `pdoc`, paths wrapped in single backticks will attempt to link to that object in the rendered documentation. Double backticks is regular inline code. I've used double backticks here (and in one other place with the same problem) for consistency.